cypress/integration/login.spec.ts

```diff
@@ -9,21 +9,15 @@ describe('Login process', () => {
 
     it('contains a login form where you can login and see your details', () => {
         cy.visit('/login');
-        cy.get('input[aria-label="E-mail address"]').type('testadmin@example.com');
-        cy.get('input[aria-label="Password"]').type('password');
-        cy.get('input[aria-label="Log in"]')
-            .should('be.visible')
-            .click();
+        cy.fillLoginForm();
         cy.url().should('match', /\/$/);
         cy.get('[data-testid="username"]').should('be.visible');
         cy.get('[data-testid="user-avatar"]').should('be.visible');
     });
 
     it('takes you to My Page when clicking on user avatar', () => {
         cy.visit('/login');
-        cy.get('input[aria-label="E-mail address"]').type('testadmin@example.com');
-        cy.get('input[aria-label="Password"]').type('password');
-        cy.get('input[aria-label="Log in"]').click();
+        cy.fillLoginForm();
```

Comment:
Would it not work to use `cy.login()` here (instead of explicitly visiting on the line before) and for other tests in this spec file as well?

It looks to me like the 'redirects to tried page…' test is the only one that would have to use `cy.fillLoginForm()` directly.

Reply:
I didn’t use cy.login() for any of the login page tests, because some tests were failing, 
since login() preserves cookies — who knows in what order any new tests might be added.